packages/angular/common/src/directives/navigation/tabs.ts

```diff
@@ -17,14 +18,15 @@ import { StackDidChangeEvent, StackWillChangeEvent } from './stack-utils';
   selector: 'ion-tabs',
 })
 // eslint-disable-next-line @angular-eslint/directive-class-suffix
-export abstract class IonTabs implements AfterContentInit, AfterContentChecked {
+export abstract class IonTabs implements AfterViewInit, AfterContentInit, AfterContentChecked {
   /**
    * Note: These must be redeclared on each child class since it needs
    * access to generated components such as IonRouterOutlet and IonTabBar.
    */
   abstract outlet: any;
   abstract tabBar: any;
   abstract tabBars: any;
+  abstract tabs: any;
```

Comment:
Minimally the type signature should be:
```ts
abstract tabs: QueryList<any>;
```

Reply:
https://github.com/ionic-team/ionic-framework/pull/29786/commits/1f8a4e38e7df518a6456ad639512930fbc456515